jsfuck.js

```diff
@@ -49,7 +49,7 @@
     'y':   '(NaN+[Infinity])[10]',
     'z':   '(+(35))["to"+String["name"]](36)',
 
-    'A':   '(+[]+Array)[10]',
+    'A':   '(NaN+[]["entries"]())[11]',
```

Comment:
can you run grunt to generate output.txt?

Reply:
Here it is:
```js
` ` 92
`!` 3558
`"` 471
`#` 3567
`$` 3572
`%` 2262
`&` 947
`'` 3587
`(` 97
`)` 97
`*` 3567
`+` 70
`,` 333
`-` 135
`.` 72
`/` 249
`0` 8
`1` 10
`2` 14
`3` 19
`4` 24
`5` 29
`6` 34
`7` 39
`8` 44
`9` 49
`:` 1677
`;` 949
`<` 237
`=` 467
`>` 243
`?` 1672
`@` 3548
`A` 185
`B` 508
`C` 2418
`D` 2578
`E` 1678
`F` 571
`G` 3425
`H` 3550
`I` 70
`J` 3556
`K` 3561
`L` 3566
`M` 3426
`N` 16
`O` 664
`P` 3554
`Q` 3556
`R` 1673
`S` 510
`T` 3428
`U` 1989
`V` 3580
`W` 3585
`X` 3559
`Y` 3561
`Z` 3565
`[` 171
`\` 1917
`]` 189
`^` 3585
`_` 3590
``` 3564
`a` 15
`b` 177
`c` 89
`d` 22
`e` 25
`f` 13
`g` 520
`h` 1219
`i` 27
`j` 182
`k` 1208
`l` 19
`m` 509
`n` 18
`o` 88
`p` 1230
`q` 950
`r` 16
`s` 24
`t` 14
`u` 16
`v` 1224
`w` 1233
`x` 1238
`y` 84
`z` 1263
`{` 92
`|` 3595
`}` 408
`~` 3605

```

Reply:
I mean commit it here in the PR.

Reply:
Sorry - I don't know how to edit another file inside pull request - i can edit only one file (I do it using github.com user interface). If you want I can create another pull request (separate to this one) in which I change output.txt

Reply:
I did that in my machine:

![image](https://user-images.githubusercontent.com/1521/92708115-2911f080-f356-11ea-9f71-5ca3030c36ac.png)

So, the code looks good. I'll merge the PR and add the output.txt later.